knative-operator/hack/update-manifests.sh

```diff
@@ -59,3 +59,7 @@ download_kafka source "${kafka_source_files[@]}"
 # artifacts are downloaded from midstream openshift/release/artifacts directory.
 # Before adding patches to this file consider sending a patch to midstream and then by just running
 # `make generated-files` the patch will appear in the final bundled artifacts.
+
+# Copy some additional manifest files that are only relevant for Serverless Operator.
+extra_dir="$root/knative-operator/deploy/resources/knativekafka_extra"
```

Comment:
why is extra a special thing?

Reply:
Because the files in `knative-operator/deploy/resources/` dir are actually downloaded from midstream within this script above. Before downloading, the script deletes the contents of the directory.

Reply:
If there's already something in midstream, I would be happy to leverage that.

Or, if you have any better idea